packages/hash/api/src/graphql/resolvers/ontology/property-type.ts

```diff
@@ -28,7 +28,9 @@ export const createPropertyType: ResolverFn<
     const msg =
       err.response?.status === 409
         ? `Property type with the same URI already exists. [URI=${propertyType.$id}]`
-        : `Couldn't create property type.`;
+        : `Couldn't create property type. ${JSON.stringify(
```

Comment:
I think it's useful to have some of the response from the Graph API relayed to the GQL endpoint response, but if we do this here, we should do it for the other ontology types as well (If committing this was intention)

Reply:
I didn't quite realise this was going through GQL (despite this being a resolver 🙃 😂 ). I've reverted the change, I think we can revisit error-reporting in general later.